### PR TITLE
Issue#45 fe landing page event card list

### PR DESCRIPTION
**Pull request by mariagluck**
Added Frontend Card component with favorite button and edited styled Landing Page

### CODE REVIEW

File: src/components/Card/text.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import moment from "moment";
```
Comment (Mxx1029) [inline src/components/Card/text.js:1]: this can be deleted, no?

File: src/components/LandingPage/LandingPage.jsx [hunk 2]
```diff
@@ -12,8 +13,22 @@ export default function LandingPage(){
                 <Header />
                 <SearchNav />
             </div>
+            <main>
+                <Card />
```
Comment (Mxx1029) [inline src/components/LandingPage/LandingPage.jsx:17]: do you plan in the future to put a Cards component, that maps the Card component? I guess that's the plan ?

**Review by Mxx1029 — APPROVED** [review]

**Comment by Mxx1029** [comment]
Great work, Maria - code looks beautiful 💯

File: src/components/LandingPage/LandingPage.jsx (hunk 2 above)
Comment (mariagluck) [reply src/components/LandingPage/LandingPage.jsx:17]: lading Page is the component that maps the Cards. the cardlist would be different depending on login or not status, favorited events-cards, and searched evens list

File: src/components/Card/text.js (hunk 1 above)
Comment (mariagluck) [reply src/components/Card/text.js:1]: haha, I thought it was!

**Pull request merged**